presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/SymbolMapper.java

```diff
@@ -81,17 +83,25 @@ public Expression rewriteSymbolReference(SymbolReference node, Void context, Exp
         }, value);
     }
 
-    public OrderingScheme map(OrderingScheme orderingScheme)
+    public RowExpression map(RowExpression value)
     {
-        return new OrderingScheme(orderingScheme.getOrderBy().stream().map(this::map).collect(toImmutableList()),
-                orderingScheme.getOrderings().entrySet().stream().collect(toMap(entry -> map(entry.getKey()), entry -> entry.getValue())));
+        if (isExpression(value)) {
+            return castToRowExpression(map(castToExpression(value)));
+        }
+        return rewriteWith(value, this::map);
     }
 
-    // TODO this will be removed later after FunctionCall is removed from aggregation
-    public OrderBy map(OrderBy orderBy)
+    public OrderingScheme map(OrderingScheme orderingScheme)
     {
-        return new OrderBy(orderBy.getSortItems().stream()
-                .map(sortItem -> new SortItem(map(sortItem.getSortKey()), sortItem.getOrdering(), sortItem.getNullOrdering())).collect(Collectors.toList()));
+        // This logic is slightly different from SymbolInliner
```

Comment:
comment not addressed (https://github.com/prestodb/presto/pull/12710#discussion_r286711623)